src/main/groovy/com/avast/gradle/dockercompose/ComposeSettings.groovy

```diff
@@ -101,7 +101,7 @@ abstract class ComposeSettings {
             else {
                 return "${safeProjectNamePrefix}_${nestedName}"
             }
-        })
+        }).map{ it.toLowerCase()}
```

Comment:
Could you please just drop a comment here, why are we lowercasing? 🙏

Reply:
Sure, done 273db55e314c83a2f385e41ec51f33db2e929b0f